ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -179,6 +179,11 @@
      * Header that is set in the response of OPTIONS request that specifies the validity of the options returned.
      */
     public static final String ACCESS_CONTROL_MAX_AGE = "Access-Control-Max-Age";
+    /**
+     * Header that is set in the response of GetBlobInfo
+     * 'true' or 'false' case insensitive; true indicates content is encrypted at the storage layer. false otherwise
+     */
+    public final static String IS_ENCRYPTED_ON_STORAGE = "x-ambry-is-encrypted-on-storage";
```

Comment:
would prefer, no `IS_` - same in the value

Reply:
"x-ambry-encrypted-*in*-storage" sounds a little better to me.